packages/react-examples/.storybook/preview.js

```diff
@@ -5,68 +5,69 @@ import 'cypress-storybook/react';
 addParameters({
-  a11y: {
+  a11y: /** @type {import('@storybook/addon-a11y').A11yParameters} */ ({
     manual: true,
-  },
+  }),
 });
 
+// This API is deprecated and will no longer work in storybook 7 or with storyStoreV7.
+// https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-configure
+// The selective loading functionality currently implemented by preview-loader can be more easily
+// handled by just settings `stories: [/*glob*/]` in main.js, but it's unclear if a replacement is
+// available for transforming files before loading as stories.
 configure(loadStories, module);
 
-export const parameters = {};
-
-// ================================
-//          Helpers
-// ================================
+// ==========================================================
+//   Helpers for loading examples in component story format
+// ==========================================================
 
 /**
  * @typedef {{
  *   default: { title: string };
- *   [subStoryName: string]: React.FunctionComponent | { title: string };
- * }} Story
+ *   [storyName: string]: React.FunctionComponent | { title: string };
+ * }} StoryGroup - Group of stories for a particular component (follows component story format)
  *
- * @typedef {{ [exportName: string]: React.ComponentType }} ComponentModule
+ * @typedef {{
+ *   [exportName: string]: React.ComponentType;
+ * }} ExampleModule - A loaded `*.Example.tsx` or `*.stories.tsx` file, exporting an example component
  */
 
-/** */
+/**
+ * Read the `*.Example.tsx` and/or `*.stories.tsx` files for the current package `start-storybook`
+ * is running in and generate a list of stories in component story format.
+ */
 function loadStories() {
-  /** @type {Map<string, Story>} */
+  /** @type {Map<string, StoryGroup>} */
   const stories = new Map();
 
-  // This shows some extra e2e-only stories
-  const includeE2E = new URLSearchParams(location.search).has('e2e');
-
   /** @type {__WebpackModuleApi.RequireContext[]} */
   const contexts = [
     // This will be updated by preview-loader with the actual current package name
     require.context('../src/PACKAGE_NAME', true, /\.(Example|stories)\.tsx$/),
   ];
 
+  // This will be replaced with the actual package name by the custom webpack loader ./preview-loader.js
+  // (note: for some reason the replacement doesn't work if the const is outside the function)
+  const packageNamePlaceholder = /** @type {string} */ ('PACKAGE_NAME');
+
   if (packageNamePlaceholder === 'react') {
-    // For suite package storybooks, also show the examples of re-exported component packages.
-    // preview-loader will replace REACT_ DEPS with the actual list.
-    contexts.push(
-      require.context('../src', true, /(REACT_DEPS|PACKAGE_NAME)\/\w+\/[\w.]+\.(Example|stories)\.(tsx|mdx)$/),
-    );
+    // For @fluentui/react's storybook, also include examples from @fluentui/react-focus
```

Comment:
why is this needed/demanded ? IMO adds more magic than necessary

Reply:
Because FocusZone is re-exported from `@fluentui/react`. At least with it hardcoded it's less magical than before.

The reason `@fluentui/react-focus` is a separate package is related to a previous iteration of convergence attempts with northstar, when the goal was to have northstar gradually start adopting/sharing `@fluentui/react` components (or something like that). FocusZone was the first thing split into a separate package for both northstar and v7/8 to use--I think @khmakoto worked on this. 

At this point, it's worth considering moving FocusZone back into `@fluentui/react`, and either getting rid of react-focus or flipping the dependency and making it re-export FocusZone from `@fluentui/react` (similar to what we did with react-date-time as part of the v8 release). The only partner I'm aware of with extremely rigid/deep path dependencies where this might break them is ODSP, and they're not on v8 yet.

Reply:
Opened an issue about this in #22456.